delivery_tnt_oca/models/tnt_request.py

```diff
@@ -42,6 +42,7 @@ def _send_api_request(
             headers = {"Content-Type": content_type}
```

Comment:
Looking over Internet, it seems you have to put it this way:

```suggestion
            headers = {"Content-Type": content_type; "charset": "UTF-8"}
```